src/components/UI/Overlay/OverlayContent/AddressBook.tsx

```diff
@@ -9,8 +9,14 @@ import { CsvUploadButton } from "../../../AddressBook/CsvUploadButton";
 import { isEmpty } from "lodash";
 import { makeEtherscanAddressURL } from "../../../../utils";
 import { vars } from "../../../../styles";
+import { OverlayContext } from "./../../../../common/contexts/OverlayContext";
 
-export const AddressBook = styled(({ ...props }: OverlayContentProps) => {
+interface AddressBookProps extends OverlayContentProps {
+  onSetNewValue?: (newValue: string) => void;
```

Comment:
```suggestion
  onAddressSelected?: (newValue: string) => void;
```

Suggest to change the naming to reflect the intention at this component level